From d48cc35053ab52f926e19714aaaee0aabf21105f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Erik=20Bot=C3=B6?= <erik.boto@pelagicore.com>
Date: Tue, 7 Oct 2014 09:58:00 +0200
Subject: [PATCH] systemd/*: unit type should be in lowercase, so use
 Type=simple

Otherwise systemd fails to parse these files with an error:
Failed to parse service type, ignoring: Simple
---
 systemd/dlt-adaptor-udp.service.cmake  |    2 +-
 systemd/dlt-example-user.service.cmake |    2 +-
 systemd/dlt-receive.service.cmake      |    2 +-
 systemd/dlt-system.service.cmake       |    2 +-
 systemd/dlt.service.cmake              |    2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/systemd/dlt-adaptor-udp.service.cmake b/systemd/dlt-adaptor-udp.service.cmake
index 0c39637..6f4e044 100644
--- a/systemd/dlt-adaptor-udp.service.cmake
+++ b/systemd/dlt-adaptor-udp.service.cmake
@@ -19,7 +19,7 @@ Description=GENIVI DLT adaptor stdin. Adaptor for forwarding received UDP messag
 Wants=dlt.service
 
 [Service]
-Type=Simple
+Type=simple
 User=genivi
 ExecStart=@CMAKE_INSTALL_PREFIX@/bin/dlt-adaptor-udp -a @DLT_ADAPTOR_UDP_APPID@ -c @DLT_ADAPTOR_UDP_CTID@ -p @DLT_ADAPTOR_UDP_PORT@
 LimitCORE=infinity
diff --git a/systemd/dlt-example-user.service.cmake b/systemd/dlt-example-user.service.cmake
index 3652292..7b7de20 100644
--- a/systemd/dlt-example-user.service.cmake
+++ b/systemd/dlt-example-user.service.cmake
@@ -19,7 +19,7 @@ Description=GENIVI DLT example user. Generate DLT messages and store them to fil
 Wants=dlt.service
 
 [Service]
-Type=Simple
+Type=simple
 User=genivi
 ExecStart=@CMAKE_INSTALL_PREFIX@/bin/dlt-example-user "Hallo from GENIVI DLT example user application"
 LimitCORE=infinity
\ No newline at end of file
diff --git a/systemd/dlt-receive.service.cmake b/systemd/dlt-receive.service.cmake
index b92c8b2..3ea770d 100644
--- a/systemd/dlt-receive.service.cmake
+++ b/systemd/dlt-receive.service.cmake
@@ -20,7 +20,7 @@ Documentation=man:dlt-receive(1)
 Wants=dlt.service
 
 [Service]
-Type=Simple
+Type=simple
 User=genivi
 ExecStart=@CMAKE_INSTALL_PREFIX@/bin/dlt-receive -o /tmp/dlt_receive_log.dlt localhost
 LimitCORE=infinity
\ No newline at end of file
diff --git a/systemd/dlt-system.service.cmake b/systemd/dlt-system.service.cmake
index f5e844b..3f5bd08 100755
--- a/systemd/dlt-system.service.cmake
+++ b/systemd/dlt-system.service.cmake
@@ -20,7 +20,7 @@ Documentation=man:dlt-system(1) man:dlt-system.conf(5)
 Wants=dlt.service
 
 [Service]
-Type=Simple
+Type=simple
 User=genivi
 ExecStart=@CMAKE_INSTALL_PREFIX@/bin/dlt-system
 WatchdogSec=@DLT_WatchdogSec@
diff --git a/systemd/dlt.service.cmake b/systemd/dlt.service.cmake
index 22279c4..21b1d04 100755
--- a/systemd/dlt.service.cmake
+++ b/systemd/dlt.service.cmake
@@ -19,7 +19,7 @@ Description=GENIVI DLT logging daemon
 Documentation=man:dlt-daemon(1) man:dlt.conf(5)
 
 [Service]
-Type=Simple
+Type=simple
 User=genivi
 ExecStart=@CMAKE_INSTALL_PREFIX@/bin/dlt-daemon
 WatchdogSec=@DLT_WatchdogSec@
-- 
1.7.9.5

